### PR TITLE
Fix for #10571 qx config is not working

### LINKED ISSUE #10571: qx config doesn't work
**Describe the bug**
Get the error:
```
goldim@goldim:~/WebstormProjects/ugpa.completer$ qx config list
Error: TypeError: Cannot read properties of null (reading 'getTargetType')
    at wrapper.__P_3_8 (/home/goldim/WebstormProjects/ugpa.completer/node_modules/@qooxdoo/framework/lib/compiler/index.js:35512:56)
    at async wrapper.run (/home/goldim/WebstormProjects/ugpa.completer/node_modules/@qooxdoo/framework/lib/compiler/index.js:35294:9)
    at async wrapper.main (/home/goldim/WebstormProjects/ugpa.completer/node_modules/@qooxdoo/framework/lib/compiler/index.js:34734:11)
```

**To Reproduce**
1. Create new project
2. Run `qx config list` or `qx config set`

**Expected behavior**
No error

**Env Info (please complete the following information):**
 - OS: Debian
 - qooxdoo framework 7.5.1

### PR DESCRIPTION
**Pull request by hkollmann**
i split the config command in sub commands so that they do work with the process implementation.

### CODE REVIEW

**Review by oetiker — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by goldim — APPROVED** [review]
Works great! Thanks!

File: source/class/qx/tool/cli/commands/config/Delete.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+   2017 Christian Boulanger
```
Comment (level420) [inline source/class/qx/tool/cli/commands/config/Delete.js:8]: Shouldn't this be Copyright "Henner Kollmann"?

**Review by level420 — APPROVED** [review]

**Review by derrell — APPROVED** [review]
Blind approval. I don't adequately understand what's going on in here. Nothing obvious seems problematic.

**Comment by cboulanger** [comment]
Some class doc comments are incorrect due to copy & paste - please check

File: source/class/qx/tool/cli/commands/config/Get.js [hunk 2]
```diff
@@ -30,7 +26,7 @@ qx.Class.define("qx.tool.cli.commands.config.Get", {
     getYargsCommand() {
       return {
         command: "get <key> [options]",
-        describe: "Gets a configuration value"
+        c
```
[the remark this answers is not in the document]
Comment (hkollmann) [reply source/class/qx/tool/cli/commands/config/Get.js:29]: yes thanks!

**Pull request merged**